apps/blockchain/lib/blockchain/block_setter.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Blockchain.BlockSetter do
```

Comment:
I'm not opposed to these coming out of `Block`, but I'm curious why did you bring this and `BlockGetter` out of `Block`?